tests/e2e/standard/test_fetch_tables.py

```diff
@@ -44,9 +46,15 @@ def test_fetch_runs_by_tag(self, environment, project):
         # wait for the cache to fill
         time.sleep(5)
 
-        runs_table = project.fetch_runs_table(tag=[tag1, tag2], progress_bar=False).to_rows()
-        assert len(runs_table) == 1
-        assert runs_table[0].get_attribute_value("sys/id") == run_id1
+        if with_query:
+            kwargs = {"query": f"(sys/tags: stringSet CONTAINS '{tag1}')"}
```

Comment:
In the UI, `CONTAINS` is not available for tags. For tags, owner, state, and stage we have `ONE OF`, but I guess that's not supported API-side. Are we OK with such inconsistency? Are the differences outlined anywhere?

To document this properly, I'd need to understand exactly how the query syntax compares to we have here: https://docs.neptune.ai/app/searching_table/#syntax

Reply:
@normandy7 

My best bet is that `ONE OF` is translated in the frontend to expression that is using `CONTAINS` instead (I didn't confirm that with anyone).
If you take a look at [nql grammar definition](https://gitlab.com/neptune.ml/neptune/-/blob/master/apps/leaderboard/src/main/resources/nql/NQLParser.g4) there is no `ONE OF` keyword so there is no way to run raw queries (which we are enabling) that are using this type of expression.

Reply:
UPDATE @normandy7 :
Yep thats what I thought. [There is frontend test that confirms that](https://gitlab.com/neptune.ml/neptune/-/blob/master/apps/frontend/src/neptune-app/libs/search-query/domain/search-query-model-converter.spec.ts#L55)

So basically there is backend NQL syntax, and frontend NQL syntax. Backend NQL does not contain `ONE OF` expression but frontend  NQL does and is later on translated to backend NQL syntax which gets rid of `ONE OF` using `CONTAINS`.

We are running queries against backend so we have to use NQL which does not contain `ONE OF` expression (and is not accurately documented)

Reply:
Thanks for finding out!

That's a bummer, though. I suppose we have to live with this, but from an end-user POV it can be annoying/confusing. CC: @parthpankajtiwary